Ira/movies-app/src/components/searchForm.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+
+
+interface ISearchFormProps {
+    searchInputValue: string;
```

Comment:
**searchInputValue** rename to searchValue or value or search. 
It is better not to keep specific to markup words in property names